modules/configuring-default-seccomp-profile.adoc

```diff
@@ -73,6 +73,25 @@ metadata:
 <1> The `restricted-v2` SCC is the SCC added by default, if your workload does not have access to a different SCC. 
 <2> Newly created pods in {product-version} will have the seccomp profile configured to `runtime/default` as mandated by the SCC. 
 
+In clusters upgraded to {product-version} all authenticated users have access to the `restricted` and `restricted-v2` SCC. 
+
+In this scenario if a workload is running on a 4.10 cluster that was admitted by the SCC `restricted`. When upgraded that workload may get admitted by `restricted-v2` if the workload is able to run with `restricted-v2`. That is because `restricted-v2` is the most restrictive SCC between `restricted` and `restricted-v2`.
+Conversely in a cluster upgraded to {product-version} with a workload that requires `privilegeEscalation: true` this workload will continue to have the `restricted` SCC available for any authenticated user. 
```

Comment:
I will put a disclaimer saying something like:

In the example below we're using privilegeEscalation as a feature that is allowed by restricted but not by restricted-v2. There are more features that were allowed by restricted and now are denied by restricted-v2.

Reply:
Hi @mvazquezc I hope what I now updated more accurately reflects the situation. Hopefully close to a LGTM so I can get this content merged.

Reply:
lgtm!